app/templates/buyers/award_details.html

```diff
@@ -7,11 +7,33 @@
 {% endblock %}
 
 {% block breadcrumb %}
-  {%
-    with items = breadcrumbs
-  %}
-    {% include "toolkit/breadcrumb.html" %}
-  {% endwith %}
+  {{ govukBreadcrumbs({
+    "items": [
+      {
+        "href": "/",
+        "text": "Digital Marketplace"
+      },
+      {
+        "href": url_for("buyers.buyer_dashboard"),
+        "text": "Your account"
+      },
+      {
+        "href": url_for("buyers.buyer_dos_requirements"),
+        "text": "Your requirements"
+      },
+      {
+        "href": url_for(
+          ".view_brief_overview",
+          framework_slug=brief['frameworkSlug'],
+          lot_slug=brief['lotSlug'],
+          brief_id=brief['id']),
+        "text": brief['title']
+      },
+      {
+        "text": "Tell us about your contract with " + pending_brief_response.supplierName
```

Comment:
Are we sure that `pending_brief_response.supplierName` will always be on the object? Worth having a default just in case (eg `the supplier`)?

Reply:
if there is no brief_response the view throws a 404 and the DB the API endpoint hits has its supplier FK set as NOT NULL (https://github.com/alphagov/digitalmarketplace-api/blob/master/app/models/main.py#L1795), so the context variable shouldn't end up not having the supplier's name value.

Reply:
We can have this as simply `Tell us about your contract`

Reply:
Changed.